src/pd/client.rs

```diff
@@ -63,6 +63,38 @@ impl RpcClient {
     pub fn get_leader(&self) -> Member {
         self.leader_client.get_leader()
     }
+
+    #[inline]
+    fn option() -> CallOption {
```

Comment:
call_option may be better?

Reply:
Sure, this eliminates ambiguity. Let me change it.